src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -150,7 +150,7 @@ public Task<Result<ProjectResponse[]>> GetProjectsAsync(CancellationToken token)
         public Task<Result<PropertyResponse[]>> GetPropertiesAsync(CancellationToken token)
         {
             // Since 2.6
-            var propertiesApiUrl = new Uri("api/properties/", UriKind.Relative);
+            var propertiesApiUrl = BuildRelativeUrl("api/properties/");
```

Comment:
Is it normal to have a trailing slash?

Reply:
It was like that before. A quick look at the api description shows it shouldn't be there, will remove.

Reply:
Actually, the API says the endpoint is "api/properties/index". I've checked and on master (before the refactoring) the endpoint is "api/properties/".